libs/common/src/sql-db-entities/fee-record/fee-record.types.ts

```diff
@@ -24,7 +24,7 @@ export type UpdateWithStatusParams = {
 };
 
 export type UpdateWithKeyingDataParams = {
-  status: FeeRecordStatus;
+  status: Extract<FeeRecordStatus, 'READY_TO_KEY' | 'RECONCILED'>;
```

Comment:
Can we use `FEE_RECORD_STATUS` constants?

Reply:
This is a type, so unfortunately we can't. For reference, the type `Extract` has the following definition: 

```typescript
/**
 * Extract from T those types that are assignable to U
 */
type Extract<T, U> = T extends U ? T : never;
```